### ISSUE TITLE
VZ ETL - CR3 PDFs scrapper errors

### ISSUE THREAD

**Issue by mateoclarke** [cityofaustin/atd-data-tech#7103]
Error described here:
https://austininnovation.slack.com/archives/C013G4RNJ01/p1629935968000700

**Comment by mateoclarke** [comment]
This error appeared to me to be a temporary issue on TXDOT's side.

I assumed it was an issue filling out the initial form to select an agency and blocked out these [lines of code](https://github.com/cityofaustin/atd-vz-data/blob/master/atd-etl/app/process_cris_cr3.py#L61-L66). That worked. But then I reverted the code to include those lines, and it also worked. So I just reactivated the DAG and the first two runs have worked without error.

TLDR; I didn't change any code and now its working again. If this happens again, I would check to make sure the TXDOT login page is working and that a manual login works after temporarily disabling the DAG.
